.github/workflows/build-linux.yml

```diff
@@ -53,8 +53,9 @@ jobs:
             # pool.sks-keyservers.net is the SKS GPG global keyserver pool
             # 15CF4D18AF4F7421 is the GPG key for the LLVM apt repository
             # This key is not in the keystore by default for Ubuntu so we need to add it.
-            gpg --keyserver pool.sks-keyservers.net --recv-key 15CF4D18AF4F7421
-            gpg -a --export 15CF4D18AF4F7421 | sudo apt-key add -
+            LLVM_KEY=15CF4D18AF4F7421
+            gpg --keyserver ipv4.pool.sks-keyservers.net --recv-key $LLVM_KEY || gpg --keyserver ha.pool.sks-keyservers.net --recv-key $LLVM_KEY
```

Comment:
Thanks for this update Fan. Do you mind adding a brief note about what ipv4 represents, vs. the pool we used beforehand?

Reply:
done